Source/BSN.IpTables.Api/Controllers/V1/HomeController.cs

```diff
@@ -42,8 +43,9 @@ public async Task<ActionResult<Response<IpTablesChainSetViewModel>>> List()
         [HttpPut]
         [Route("Insert")]
         [ProducesResponseType(typeof(Response), (int)HttpStatusCode.OK)]
-        public async Task<ActionResult<Response>> Insert([FromQuery] RulesCommandServiceInsertRequest request)
+        public async Task<ActionResult<Response>> Insert([FromQuery] RulesCommandServiceInsertRequest insert_request)
```

Comment:
Why rename input parameters?

please add implementation command for describe this

(into the method, not above the method)

Reply:
Input parameter name could not be "request" because it cause error in Autorest code generation. I added this as a comment in first controller method.